pkg/envoy/suite_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestEndpoints(t *testing.T) {
+func TestEnvoySds(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Test Suite")
+	RunSpecs(t, "Envoy SDS Test Suite")
```

Comment:
Seems like we should omit SDS here.

Reply:
There are a bunch of discovery services for Envoy and their tests are named as EnvoyRds, EnvoyEds, EnvoySds. So I'd prefer if we have this as is

Reply:
I see the pkg/envoy/sds package is named like this which makes sense, but it's not clear to me why pkg/envoy should also be named this way.

Reply:
You are right, I missed that this was for pkg/envoy. I have updated the name